pkg/providers/instancetype/instancetype.go

```diff
@@ -298,11 +295,11 @@ func memoryMiB(sku *skewer.SKU) int64 {
 	return int64(memoryGiB(sku) * 1024)
 }
 
-func memory(ctx context.Context, sku *skewer.SKU) *resource.Quantity {
+func memory(sku *skewer.SKU, vmMemoryOverheadPercent float64) *resource.Quantity {
 	memory := resources.Quantity(fmt.Sprintf("%dGi", int64(memoryGiB(sku))))
 	// Account for VM overhead in calculation
 	memory.Sub(resource.MustParse(fmt.Sprintf("%dMi", int64(math.Ceil(
-		float64(memory.Value())*options.FromContext(ctx).VMMemoryOverheadPercent/1024/1024)))))
```

Comment:
does this come from core?

Reply:
Not core but AWS provider: https://github.com/aws/karpenter-provider-aws/blob/531d7e00ad69bed8fa74de79abdaf8d61b476910/pkg/providers/instancetype/types.go#L224.

Reply:
But, yes, this will introduce a difference between the two.
In general, I think our changes it will be for the better (at least, intention-wise). But the trade-off is consistency.

In this case, I am not completely sure, but leaning towards keeping current change.
Current changes mean consistency with this pattern within our provider (e.g., providers).
Restoring to original means consistency with the AWS provider. But regardless whether we keep other changes or restore all, inconsistency remains within our provider, unless we become more extreme with the use of `ctx`.